### PR TITLE
moved response and wavelength to properties to avoidn accidental setting

### PR DESCRIPTION
**Pull request by grburgess**
I need access to wavelength without having to call a private member ;)
I also moved response to a property

### CODE REVIEW

**Comment by dkirkby** [comment]
I don't remember why I didn't create a wavelength property earlier, but I don't see any reason not to.

Can you add (numpy-style) docstrings for the two new properties?  You can test new docs with:
```
cd docs
make html
open _build/html/index.html
```

**Comment by grburgess** [comment]
I added but it does seem to be popping up in the docs.

File: speclite/filters.py [hunk 1]
```diff
@@ -653,6 +653,10 @@ class FilterResponse(object):
         Linear interpolator of our response function that returns zero for
         all values outside our wavelength range.  Should normally be evaluated
         through our :meth:`__call__` convenience method.
+    wavelength : numpy.ndarray
+        Numpy array of wavelenghts passed to our constructor with their
```
Comment (grburgess) [inline speclite/filters.py]: doh! cool I removed.

**Comment by dkirkby** [comment]
Thanks, merging now.

**Comment by grburgess** [comment]
Thanks! I have one more addition in a few moments... no rush!

**Pull request merged**